File: backend/models/team.js
const mongoose = require('mongoose');

const teamSchema = new mongoose.Schema({
  name : {type: String, required: true},
  mail: {type: String, required: true},
  phone: {type: String, required: true},
  image: {type: String},
  city: {type: String, required: true}
});

module.exports = mongoose.model('Team', teamSchema ,'teams'); 

File: mobile/lib/model/team.dart
import 'package:flutter/material.dart';

class Team {
  final String id;
  final String name;
  final String mail;
  final String city;
  final String phone;
  final String image;


  const Team({
    required this.id,
    required this.name,
    required this.mail,
    required this.city,
    required this.phone,
    required this.image,
  });


  factory Team.fromJson(Map<String, dynamic> json) {
    return Team(
      id: json['id'],
      name: json['name'],
      image: json['image'],
      mail: json['mail'],
      city: json['city'],
      phone:json['phone'],
    );
  }
}
